src/EditorFeatures/CSharpTest/Diagnostics/GenerateMethod/GenerateDeconstructMethodTests.cs

```diff
@@ -335,6 +335,32 @@ internal void Deconstruct(out int x, out int y)
     }
 }");
         }
+        [Fact]
```

Comment:
We leave a blank line between methods

```suggestion

        [Fact]
```